repos/PredictionIO/e2/src/main/scala/io/prediction/e2/engine/BinaryVectorizer.scala

```diff
@@ -52,7 +52,7 @@ object BinaryVectorizer {
               .filter(e => properties.contains(e._1))
               .distinct
               .collect
-              .zipWithIndex: _*
+              .zipWithIndex: _*,
```

Comment:
In the error case, when you've accidentally added or uncommented an additional arg, the error message with the comma might be slightly better.